src/Components/Navbar/Navbar.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import NavbarMenu from './Components/NavbarMenu';
+import NavbarUser from './Components/NavbarUser';
+
+import '../Navbar/Navbar.scss'
+class Navbar extends Component {
```

Comment:
```suggestion
import React, { Component } from 'react';
import { Link } from 'react-router-dom';
import NavbarMenu from './Components/NavbarMenu';
import NavbarUser from './Components/NavbarUser';
import '../Navbar/Navbar.scss'

class Navbar extends Component {
```
구분되어야 하는 것끼리 잘 구분해주세요.